graphql/task_resolver.go

```diff
@@ -522,14 +522,14 @@ func (r *taskResolver) Pod(ctx context.Context, obj *restModel.APITask) (*restMo
 func (r *taskResolver) Project(ctx context.Context, obj *restModel.APITask) (*restModel.APIProjectRef, error) {
 	pRef, err := data.FindProjectById(*obj.ProjectId, true, false)
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding project ref for project %s: %s", *obj.ProjectId, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project ref for project %s: %s", *obj.ProjectId, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project ref for project '%s': %s", *obj.ProjectId, err.Error()))
```